### PR TITLE
World wide organisations listing page to GOVUK design system.

### PR DESCRIPTION
**Pull request by farahTW**
This PR transits world wide organisations page to GDS.
It does the following:

- Duplicated existing index view and controller test to legacy.
- Added new index view with GDS.
- Added confirm destroy functionality for index view.
- Added new unit and feature tests.
- Fixed existing tests.

**Screen Shots:**
**Before:**
image

**After:**
**Index page:**
image
**Confirm destroy page:**
image
**No organisation created case:**
image

**Design review changes:
Messages**:
image

**Added country column**
image

**Trello:**
https://trello.com/c/1zhTXNQf/314-listing-page

### CODE REVIEW

**Comment by nnagewad** [comment]
I have some updates:
- Destroy page --> change headline to "Delete worldwide organisation"
- As per Slack thread, lets add a column for country
- With the filterability of organization and now country, we should update the label for the filter to "Filter by organisation or country"
- [Link to Figma](https://www.figma.com/file/bbg7C8tPnPBc0ATsNexKgd/Whitehall-transition-rough-ideas?type=design&node-id=1328%3A34834&mode=design&t=ojhruDop3sal0sSn-1)

File: app/controllers/admin/worldwide_organisations_controller.rb [hunk 1]
```diff
@@ -49,13 +52,27 @@ def set_main_office
...
   def destroy
     @worldwide_organisation.destroy!
     respond_with :admin, @worldwide_organisation
+    flash[:notice] = "Organisation deleted successfully"
   end
 
 private
 
+  def get_layout
+    design_system_actions = []
```
Comment (davidgisbey) [inline app/controllers/admin/worldwide_organisations_controller.rb:66]: confirm destroy is always in the GOV.UK Design System so i think this should be

```
    design_system_actions = %w[confirm_destroy]
 design_system_actions += %w[index if preview_design_system?(next_release: false)

```

File: app/views/admin/worldwide_organisations/index.html.erb [hunk 2]
```diff
@@ -1,41 +1,50 @@
...
+          label: "Filter by organisation or country",
+          head: [
+            {
+              text: "Organisation name"
+            },
+            {
+              text: "Country"
+            },
+            {
+              text: "",
```
Comment (davidgisbey) [inline app/views/admin/worldwide_organisations/index.html.erb:25]: Can we make this a visually hidden span for screen readers please

```
 text: tag.span("Actions", class: "govuk-visually-hidden")
```

File: app/views/admin/worldwide_organisations/index.html.erb [hunk 3]
```diff
@@ -1,41 +1,50 @@
...
-                none
-              <% end %>
-          </td>
-          <td><%= worldwide_organisation.world_locations.map {|l| link_to(l.name, [:admin, l.world_location_news])}.to_sentence.html_safe %></td>
-          <td><%= worldwide_organisation.sponsoring_organisations.map {|o| link_to(o.acronym, [:admin, o])}.to_sentence.html_safe %></td>
-        <% end %>
-      <% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+  <%= render "govuk_publishing_components/components/button", {
```
Comment (davidgisbey) [inline app/views/admin/worldwide_organisations/index.html.erb:7]: this needs to be indented 2 more please

**Comment by davidgisbey** [comment]
It's looking good. I've just left a few minor comments

File: app/views/admin/worldwide_organisations/index.html.erb [hunk 4]
```diff
@@ -1,41 +1,50 @@
...
-              <% else %>
-                none
-              <% end %>
-          </td>
-          <td><%= worldwide_organisation.world_locations.map {|l| link_to(l.name, [:admin, l.world_location_news])}.to_sentence.html_safe %></td>
-          <td><%= worldwide_organisation.sponsoring_organisations.map {|o| link_to(o.acronym, [:admin, o])}.to_sentence.html_safe %></td>
-        <% end %>
-      <% end %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
```
Comment (davidgisbey) [inline app/views/admin/worldwide_organisations/index.html.erb:6]: sorry one last thing, generally we've used full width for tables with 3 or more columns. Just wanted to double check this is fine. Probs a question for Nik though.

File: app/views/admin/worldwide_organisations/index.html.erb (hunk 2 above)
Comment (farahTW) [reply app/views/admin/worldwide_organisations/index.html.erb:25]: yes i have done that already just need to push it

**Review by davidgisbey — APPROVED** [review]
lgtm!

**Pull request merged**